src/OrchardCore/OrchardCore.AsBundle/Bundle.Build.targets

```diff
@@ -21,6 +21,13 @@
   -->
 
   <ItemGroup>
+    <Compile Remove="App_Data\**;Packages\**" />
```

Comment:
You're right, App_data should only appear in Orchard,Cms.Web (or another user app). But just below we also use App_Data for a custom publish because we don't want that App_Data appears in the solution.

So, if we also let the web app define App_Data to be published in a regular way, App_Data will appear in the solution. If you think it's not a problem, we can only use App_Data in Orchard.Cms.Web and here just use Packages.

If so, do i keep here a custom publish for Packages? Or better to revert back to a regular one even if Packages will appear in the solution.